graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -120,9 +123,9 @@ public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProp
                         sourceInput.processRawMessage(rawMessage);
                         channel.basicAck(deliveryTag, false);
                     } catch (Exception e) {
-                        LOG.error("Error while trying to process AMQP message, requeuing message", e);
+                        LOG.error("Error while trying to process AMQP message", e);
                         if (channel.isOpen()) {
-                            channel.basicNack(deliveryTag, false, true);
+                            channel.basicNack(deliveryTag, false, requeueInvalid);
```

Comment:
Maybe add a log message on debug level whether the message was re-queued or not (including the delivery tag).